sandbox/pages/SandboxToaster.vue

```diff
@@ -5,9 +5,20 @@
   >
     <div class="ktoaster-sandbox">
       <SandboxSectionComponent>
-        <KExternalLink href="https://www.figma.com/file/Yze0SWXl5nKjR0rFdilljK/Components?type=design&node-id=497%3A9761&mode=dev&t=lcG4fMxcpZoeYwsC-1">
-          Figma
-        </KExternalLink>
+        <div>
+          <KExternalLink href="https://www.figma.com/file/Yze0SWXl5nKjR0rFdilljK/Components?type=design&node-id=497%3A9761&mode=dev&t=lcG4fMxcpZoeYwsC-1">
```

Comment:
we don't want figma links in the sandbox since they're not public (I thought we already removed all these?)

Reply:
I will handle it in a separate PR 👍